packages/datetime/src/dateRangePicker.tsx

```diff
@@ -197,13 +206,22 @@ export class DateRangePicker extends AbstractComponent<IDateRangePickerProps, ID
 
     public render() {
         const modifiers = combineModifiers(this.modifiers, this.props.modifiers);
-        const { className, contiguousCalendarMonths, locale, localeUtils, maxDate, minDate } = this.props;
+        const {
+            className,
+            contiguousCalendarMonths,
+            dayPickerProps,
+            locale,
+            localeUtils,
+            maxDate,
+            minDate,
+        } = this.props;
         const isShowingOneMonth = DateUtils.areSameMonth(this.props.minDate, this.props.maxDate);
 
         const { leftView, rightView } = this.state;
         const disabledDays = [{ before: this.props.minDate }, { after: this.props.maxDate }];
 
-        const dayPickerBaseProps: DayPicker.Props = {
+        const hydratedDayPickerProps: ReactDayPicker.Props = {
```

Comment:
Rename to `baseProps`. "hydrated" is nifty but is overly verbose and appears nowhere else in our codebase.